### PR TITLE
Add TD Canada Trust to Banking

### PR DESCRIPTION
**Pull request by ESWAT**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -285,6 +285,12 @@ websites:
       img: tdbank.png
       tfa: No
 
+    - name: TD Canada Trust
+      url: http://www.tdcanadatrust.com
```
Comment (Carlgo11) [inline _data/banking.yml]: The site supports https. Please change it to `https://www.tdcanadatrust.com`

**Comment by Carlgo11** [comment]
Wow we must have missed this PR. Can I get a second opinion on these additions so that we can merge quickly.
:open_mouth:

**Comment by Carlgo11** [comment]
There we go.
Sorry for the delay @ESWAT.

Thank you for the contribution :smiley:

**Comment by ESWAT** [comment]
No worries. To be fair the tests failed originally - or never finished to be more accurate - so I just opened/closed the PR to re-run the tests knowing a failed PR wasn’t going to get merged. :)

**Pull request merged**